src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -275,7 +276,7 @@ public string RootId
         /// Returns the value of the key-value pair added to the activity with <see cref="AddBaggage(string, string)"/>.
         /// Returns null if that key does not exist.
         /// </summary>
-        public string GetBaggageItem(string key)
+        public string? GetBaggageItem(string? key)
```

Comment:
Per this method, it seems like the Baggage `KeyValuePair` should be `KeyValuePair<string?, string?>`

Reply:
you are right i have confused with KeyValuePair generics, but this method return and accept null no matter of KeyValuePair generics